src/nav-soknad/containers/StegMedNavigasjon.tsx

```diff
@@ -154,13 +154,15 @@ const StegMedNavigasjon = (
         }
     };
 
-    const nextButtonPending = soknad.sendSoknadPending || isAdresseValgRestPending;
     const {skjemaConfig, stegKey, ikon, children} = props;
 
     const {feil, visValideringsfeil} = validering;
 
     const aktivtStegConfig: SkjemaSteg | undefined = skjemaConfig.steg.find((s) => s.key === stegKey);
 
+    const nextButtonPending =
+        soknad.sendSoknadPending || aktivtStegConfig?.key === "kontakt" ? isAdresseValgRestPending : false;
```

Comment:
Noen steder bruker vi 
aktivtStegConfig.stegnummer
andre steder: 
aktivtStegConfig.type

Noen grunn til at du gikk for 
aktivtStegConfig?.key

Dette er noe vi kan forsøke å få konsekvent senere. 
(Godkjenner forde altså)

Reply:
Landet kun på den siden den var deklarert tidligere, og det var ganske intuitivt at dette var det aktive steget. Ellers helt enig i at vi bør rydde litt i dette etterhvert. :)